src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,95 @@
+import "./Sidebar.css";
+import { useProduct } from "../../context/product-context";
+export const SideBar = () => {
+  const { selfhelp, business, biography, sortBy, spirtual, productsDispatch } =
+    useProduct();
+
+  return (
+    <aside className="sidebar">
+      <p className="filter-text">filters</p>
+      <button className="filter-btn">clear</button>
+      <div className="sidebar-sortby">
+        <div className="sortby-heading">sort by</div>
+        <label className="sortby-label" htmlFor="high-to-low">
+          <input
+            type="radio"
+            id="high-to-low"
+            name="sortBy"
+            checked={sortBy === "HIGH_TO_LOW"}
+            onChange={() => productsDispatch({ type: "HIGH_TO_LOW" })}
+          />
+          Price : Hight to Low
+        </label>
+        <label className="sortby-label" htmlFor="low-to-high">
+          <input
+            type="radio"
+            id="low-to-high"
+            name="sortBy"
+            checked={sortBy === "LOW_TO_HIGH"}
+            onChange={() => productsDispatch({ type: "LOW_TO_HIGH" })}
+          />
+          Price : Low to High
+        </label>
+      </div>
+      <div className="sidebar-rating">
+        <div className="rating-heading">rating</div>
+        <label className="rating-label" htmlFor="four">
+          <input type="radio" id="four" name="rating" />4 star & above
```

Comment:
This is repetative, you can use an array and map it based on the values. 

The structure would be like

```
const filters = {
   rating : [
      {
        name: 4star and above
       },
      {
        name: 3 star and above
       } 
   ]
}
```
and you can map it like this

```
filters.rating.map((item) => <label className="rating-label" htmlFor="three">
          <input type="radio" id="three" name="rating" />{item.name}
        </label>)
```

Reply:
done